servant-client/servant-client.cabal

```diff
@@ -46,9 +56,14 @@ library
     , text
     , transformers
     , transformers-compat
+    , case-insensitive
+  if impl(ghcjs)
+    build-depends:
+        ghcjs-base
+      , ghcjs-prim
   hs-source-dirs: src
   default-language: Haskell2010
-  ghc-options: -Wall
+  ghc-options: -Wall -Werror
```

Comment:
Removed it. We don't have `-Werror` on travis, I think we also have some warnings in some servant packages. (https://github.com/haskell-servant/servant/pull/273)